openmeetings-webservice/pom.xml

```diff
@@ -56,6 +56,41 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+		      <groupId>io.openapitools.swagger</groupId>
```

Comment:
there seems to be whitespace issue
please use tabs only

Reply:
Done